app/views/accounts/_connected_app.html.erb

```diff
@@ -8,8 +8,9 @@
           <%= identity.display_name %>
         <% end %>
       </div>
+
       <%= t('account.connected_apps.associated',
-            timestamp: local_time(identity.created_at, t('time.formats.event_timestamp'))).html_safe %>
+            timestamp: local_time(identity.created_at.utc, t('time.formats.event_timestamp'))).html_safe %>
```

Comment:
```suggestion
            timestamp: local_time(identity.created_at.in_time_zone('UTC'), t('time.formats.event_timestamp'))).html_safe %>
```

So `#utc` mutates the underlying timestamp object (booo), I think ActiveSupport's `#in_time_zone` would be a good alternative?